pkg/cli/gettoken/gettoken.go

```diff
@@ -241,7 +242,7 @@ func (o *GetTokenOptions) doAuthCode(ctx context.Context) (string, string, error
 			}
 
 			if !o.AutoOpenBrowser {
-				fmt.Fprintf(o.IOStreams.Out, "Please visit the following URL in your browser: %s", url)
+				fmt.Fprintf(o.IOStreams.ErrOut, "Please visit the following URL in your browser: %s\n", url)
```

Comment:
the original seems better, maybe?

Reply:
`IOStreams.Out` is listened by client-go to get the tokens, we have to write to `ErrOut`.

Reply:
Interesting. Please comment.